scss/_patterns_icons.scss

```diff
@@ -203,7 +204,7 @@ $color-social-icon-foreground: $color-mid-x-light !default;
     @include vf-icon-anchor($color-mid-dark);
 
     [class*='--dark'] &,
-    &.is-light {
+    &.is-white {
```

Comment:
these renames  are a mistake, should be is-light.

Reply:
@anthonydillon do you agree with treating f7f7f7 as its own theme and reserving the name for it?